source/chapters/user_interface.rst

```diff
@@ -551,41 +565,38 @@ Loop Controls
 In this section of the Mixxx interface you can control (beat-)loops and set the
 loop points of a track.
```

Comment:
```suggestion
loop points of a track.
You can also shift loops using the :ref:`beatjump / loop move controls <transport-controls>` shown at the right side of the looping controls.
```

Reply:
Adding this change gives me this warning when i run `sphinx-build source build/html -b html -d build/doctrees -c source`

![image](https://user-images.githubusercontent.com/28389950/97626677-3e90c600-1a3b-11eb-8778-e66a6bfaa928.png)

Reply:
oh my bad, wanted to help but pasted the wrong string.
it needs to ref to Transport Controls. the ref is line 394 `interface-transport`.
I adapted the code suggestion accordingly.